### PR TITLE
set content offset on end refresh

### PR DESCRIPTION
**Pull request by cjc8**

### CODE REVIEW

File: LUX/Classes/TableViews/View Models/LUXTableViewModels.swift [hunk 1]
```diff
@@ -48,18 +48,21 @@ extension LUXRefreshableTableViewModel {
...
         publisher.$error.sink { _ in
             DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                self.tableView?.setContentOffset(.zero, animated: true)
                 self.endRefreshing()
             }
         }.store(in: &cancelBag)
         
         publisher.$serverError.sink { _ in
             DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                self.tableView?.setContentOffset(.zero, animated: true)
```
Comment (schrockblock) [inline LUX/Classes/TableViews/View Models/LUXTableViewModels.swift:65]: This is great, although I'm realizing now that this whole section is a little verbose — want to make a function that ends refreshing which these publishers call from `runOnMain`?

**Review by schrockblock — APPROVED** [review]
This looks great, just one way we could improve this whole section of functionality!

File: LUX/Classes/TableViews/View Models/LUXTableViewModels.swift (hunk 1 above)
Comment (cjc8) [reply LUX/Classes/TableViews/View Models/LUXTableViewModels.swift:65]: I'm a little confused, what would the function take in as parameters? UITableVIew, the entire model?

Comment (cjc8) [reply LUX/Classes/TableViews/View Models/LUXTableViewModels.swift:65]: Or would it be just a class function

Comment (schrockblock) [reply LUX/Classes/TableViews/View Models/LUXTableViewModels.swift:65]: It could just be a method on the object, doesn't need to be stand alone

**Review by schrockblock — APPROVED** [review]
LGTM

**Pull request merged**